build.gradle

```diff
@@ -707,8 +707,35 @@ class JDKDetails {
     }
 
     String createDownloadUrl() {
+        return createElasticCatalogDownloadUrl()
+    }
+
+    private String createElasticCatalogDownloadUrl() {
+        // Ask details to catalog https://jvm-catalog.elastic.co/jdk/latest_adoptiumjdk_17_windows
+        // and return the url to download the JDK
+
+        // arch x86_64 never used, only aarch64 if macos
+        def url = "https://jvm-catalog.elastic.co/jdk/latest_adoptiumjdk_${major}_${osName}"
```

Comment:
doing this locally it seems the url gets enhanced with AWSAccessKeyId and x-amz-security-token, I don't think we should be logging this, though it's coming from the download library.

```
> Task :downloadJdk
Download https://s3.us-west-2.amazonaws.com/jvm-catalog.elastic.co/adoptiumjdk-17.0.9_9-windows.zip?AWSAccessKeyId=XXXXXX&Signature=YYYYYYY&x-amz-security-token=ZZZZZZZZ&Expires=1698670283
Downloaded to /Users/joaoduarte/elastic/logstash/build/jdk-17.0.9-windows-x64.zip
```

Reply:
Fixed by 375503847180708c3772ed8c6cc8ca022021a626